packages/story-editor/src/components/panels/shared/captionsPanelContent.js

```diff
@@ -184,6 +184,7 @@ function CaptionsPanelContent({
                 allowedFileTypes={allowedFileTypes}
                 insertText={__('Use caption', 'web-stories')}
                 insertingText={__('Selecting caption', 'web-stories')}
+                canUseProxy={false}
```

Comment:
What is the point of the property if it is always false?

Reply:
It default's to `true` in the component. But for captions it's disabled.